de-de/README.md

```diff
@@ -0,0 +1,39 @@
+# 🇩🇪 Deutsch (Deutschland)
+
+Willkommen beim Fabulously Optimized Wiki!
+
+[⬇️ Modpack installieren](de-de/einrichtung.md) | [🆕 Modpack aktualisieren](de-de/update-anleitung.md) [(Einstellungen zurücksetzen)](de-de/update-anleitung.md#einstellungen-zurücksetzen) | [➕ Mods hinzufügen](de-de/mods-hinzufuegen.md) | [➖ Mods deaktivieren](de-de/mods-deaktivieren.md) | [🎛️ Geänderte Optionen](de-de/geaenderte-optionen.md) [(behobene Fehler)](de-de/geaenderte-optionen.md#behobene-fehler) [(wie konfiguriere ich Mods?)](de-de/geaenderte-optionen.md#mods-konfigurieren)
```

Comment:
All readme links are broken as readme.md is on the same folder as those pages - remove `de-de/` from the links
(I guess it was correct for you before when you placed the readme to root folder.)

Reply:
Okay!